Frameworks/CoreText/CTTypesetter.mm

```diff
@@ -127,13 +128,23 @@ static CFIndex _DoWrap(CTTypesetterRef ts, CFRange range, WidthFinderFunc widthF
 
         if (chars[curIndex] == 10 || (curIndex > 0 && chars[curIndex - 1] == 13 && curIndex - 1 >= lineStart)) {
             if ((curIndex > 0 && chars[curIndex - 1] == 13 && chars[curIndex] != 10)) {
-                curIndex--;
+                --curIndex;
+            }
+
+            if (curIndex > lineStart && chars[curIndex - 1] == ' ') {
+                // Ending line with trailing spaces, only want width up to last printable glyph
+                lineWidth = ((float)lastPossibleBreakWidth) / 64.0f;
```

Comment:
I'm not totally sure, which is why I didn't mess with it, but it seems like a scaling conversion.

---
In reply to: [74635158](https://github.com/Microsoft/WinObjC/pull/873#discussion_r74635158) [](ancestors = 74635158)